Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -648,17 +648,19 @@ def get_mapping_fields_command() -> GetMappingFieldsResponse:
     return mapping_response
 
 
-def close_incident_in_remote(delta: Dict[str, Any]) -> bool:
+def close_incident_in_remote(delta: Dict[str, Any], data: Dict[str, Any]) -> bool:
     """
     Closing in the remote system should happen only when both:
         1. The user asked for it
-        2. The closing field is in the delta
+        2. The incident current status is not closed.
+        3. A closing reason was provided (either in the delta or before in the data).
 
     The second is mandatory, so a closing request will not be sent for all mirroring requests that occur after closing an incident
     (in the case where the incident has been updated, but the status has not been changed).
     """
     closing_field = 'classification'
-    return demisto.params().get('close_ticket') and closing_field in delta
+    closing_reason = delta.get(closing_field, data.get(closing_field, ''))
+    return demisto.params().get('close_ticket') and data.get('status', '') != 'Closed' and bool(closing_reason)
```

Comment:
We've added a new condition `data.get('status', '') != 'Closed'` why is it necessary?

Reply:
So we won't send a closing request to an already closed ticket (every time any other field is updated). If I understood the function documentation correctly, this is why the closing field was checked in the delta in the first place.